### DISCUSSION TITLE
ogb version in submission

### DISCUSSION THREAD

**Discussion by zoexu119** [snap-stanford/ogb#392] [PCQM4M-LSC]
Hi!

We're using ogb 1.3.4. However, when we submit the test result and write ogb 1.3.4 in the submission form, it shows the error "Dataset and version mismatch".  If we write ogb 1.3.3 in the submission form, we can submit it successfully. We're worried about why such an error occurs. Are we allowed to use ogb 1.3.4? Can we just write ogb 1.3.3 in the submission form?

Thank you!

**Comment by weihua916** [comment]
Hi! Thanks for letting us know! 

1.3.4 is fine. We've just fixed the issue. 1.3.4 should be acceptable now. Let us know if it works.

**Reply by zoexu119** [reply]
Thank you very much, Weihua! 

BTW, are we expected to receive a confirmation email once the submission is received? We can see a "Sucess!" message on the website but don't receive any confirmation email.

**Reply by weihua916** [reply]
You are welcome! No you won't receive an email. As long as you see "Success!", you will be good!

**Reply by zoexu119** [reply]
Great! Thank you for your quick response!
